### PR TITLE
[REF] product_supplierinfo_for_customer: Inherit product name_search to use it in sale and purchase

### PR DESCRIPTION
**Pull request by ysantiago**

### CODE REVIEW

**Comment by moylop260** [comment]
@aheficent Do looks better for you?

**Review by AaronHForgeFlow — COMMENTED** [review]
It's not working for me, looking for products for a customer in the sales order line. The customer's product code and customer's product name are not showing.

File: product_supplierinfo_for_customer/__openerp__.py [hunk 1]
```diff
@@ -17,20 +17,24 @@
...
+              "AvanzOSC, "
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Vauxoo, "
               "Odoo Community Association (OCA)",
     "website": "http://www.odoomrp.com",
     "category": "Sales Management",
     "license": 'AGPL-3',
     "depends": [
-        "base",
+        "sale",
```
Comment (pedrobaeza) [inline product_supplierinfo_for_customer/__openerp__.py:30]: You shouldn't add dependencies to current module. Create new modules `product_supplierinfo_for_customer_sale` and ...purchase with that purpose, or use new module https://github.com/OCA/server-tools/pull/876 for conditional dependencies

**Comment by moylop260** [comment]
Superseded by https://github.com/OCA/product-attribute/pull/280

**Pull request closed without merging**